### PR TITLE
Us24 s06 art validation

### PR DESCRIPTION
**Pull request by Alice-Berthelot**

### CODE REVIEW

**Review by sorrento974 — APPROVED** [review]

File: client/src/pages/Validation.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { Link, useLoaderData } from "react-router-dom";
+import BtnGoBack from "../components/BtnGoBack";
+import DesktopBar from "../components/DesktopBar";
+import "../styles/Validation.css";
+
+function Validation() {
+  const comparedArts = useLoaderData();
+
```
Comment (Dinosaure7) [inline client/src/pages/Validation.jsx:8]: delete space

File: client/src/pages/Validation.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const pendingArts = comparedArts.filter((art) => art.status === "pending");
+
```
Comment (Dinosaure7) [inline client/src/pages/Validation.jsx:10]: delete space

File: client/src/pages/Validation.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  const artUrl = import.meta.env.VITE_API_URL;
+
```
Comment (Dinosaure7) [inline client/src/pages/Validation.jsx:12]: delete space

File: client/src/pages/ValidationDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
+import { Form, useLoaderData, useParams } from "react-router-dom";
+import { useState } from "react";
+import BtnGoBack from "../components/BtnGoBack";
+import DesktopBar from "../components/DesktopBar";
+import "../styles/Validation.css";
+import arrowDown from "../assets/images/doublearrowdown.svg";
+
+function ValidationDetails() {
+  const comparedArts = useLoaderData();
+
```
Comment (Dinosaure7) [inline client/src/pages/ValidationDetails.jsx:10]: delete space

**Review by Dinosaure7 — APPROVED** [review]

**Review by Delgorithm — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**